tests/cases/standalone/common/create/create_database.result

```diff
@@ -0,0 +1,19 @@
+create database illegal-database;
```

Comment:
I just noticed that we don't support `drop database` yet? Shall we do that?

Reply:
This feature has not been implemented yet. Issue: https://github.com/GreptimeTeam/greptimedb/issues/825